weareinreach/inreach-catalog
# Feature/1202422153673012/chinese to mandarin

## Description
change the catalog so Mandarin = Mandarin on the front end (catalog app)

Made a change to src/utils/propertyMap.js:

change:
'lang-mandarin': {code: 'zh'},

to
'lang-mandarin': {name: 'Mandarin'},

## Asana ticket:
https://app.asana.com/0/1132189118126148/1202422153673012

## PR Checklist




## How to Test
Search for an organization that contains "lang-mandarin" and search for the same organization in the catalog and it should display the language as "mandarin" instead of previously "chinese."
